pkg/server/server_controller_http.go

```diff
@@ -238,9 +238,17 @@ func (c *serverController) attemptLogoutFromAllTenants() http.Handler {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		sessionCookie, err := r.Cookie(SessionCookieName)
+		sessionCookie, err := r.Cookie(MultitenantSessionCookieName)
+		if errors.Is(err, http.ErrNoCookie) {
+			sessionCookie, err = r.Cookie(SessionCookieName)
```

Comment:
Is it definitely impossible to have both cookies set? I assume so.

Reply:
It is **not** impossible if you run mixed versions on `localhost` for instance. Or upgrade to a new version and have a session lingering from an old version.

Reply:
Generally we should be tolerant of web developers manually editing cookies in their web browser for testing/experimentation. The code should ensure we fall back to one of the known  situations in that case.

Reply:
Gotcha. My question was actually based on a misreading of the code. Even if we have both cookies, logout should still work here since we clear both cookies down below unconditionally.